Tools/CISupport/ews-build/steps.py

```diff
@@ -6176,6 +6196,7 @@ def evaluateCommand(self, cmd):
                 if self.getProperty('github.number', ''):
                     self.build.addStepsAfterCurrentStep([
                         CleanGitRepo(),
+                        SetCredentialHelper(),
```

Comment:
Do you actually want to re-run `SetCredentialHelper ` step if `PushCommitToWebKitRepo` fails?